system/Commands/Translation/LocalizationFinder.php

```diff
@@ -0,0 +1,370 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Commands\Translation;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use Config\App;
+use Locale;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use SplFileInfo;
+
+class LocalizationFinder extends BaseCommand
+{
+    protected $group       = 'Translation';
+    protected $name        = 'lang:find';
+    protected $description = 'Find and save available phrases to translate.';
+    protected $usage       = 'lang:find [options]';
+    protected $arguments   = [];
+    protected $options     = [
+        '--locale'   => 'Apply changes to the selected locale (en, ru, etc...).',
+        '--dir'      => 'Directory for searching for translations (in relation to the APPPATH).',
+        '--show-new' => 'See only new translations as table.',
+        '--verbose'  => 'Output detailed information.',
+    ];
+
+    /**
+     * Flag for output detailed information
+     */
+    private bool $verbose = false;
+
+    /**
+     * Flag for showing only translations, without saving
+     */
+    private bool $showNew = false;
+
+    private string $languagePath;
+
+    public function run(array $params)
+    {
+        $this->verbose      = array_key_exists('verbose', $params);
+        $this->showNew      = array_key_exists('show-new', $params);
+        $optionLocale       = ! empty($params['locale']) ? $params['locale'] : null;
+        $optionDir          = ! empty($params['dir']) ? $params['dir'] : null;
```

Comment:
Do you really need to use `empty()`?
```suggestion
        $optionLocale       = $params['locale'] ?? null;
        $optionDir          = $params['dir'] ?? null;
```

Reply:
Yes, i can shorten it. It seems I've worked with other options, but haven't changed for here.